index.html

```diff
@@ -28,6 +28,31 @@ <h1>Iheagwara<br>Augustine</h1>
         <button class="Start-collab">Start collaboration</button>
         </section>            
         <div class="image-1-wrapper"><div class="image-1"></div></div>
+        <!-- desktop version -->
+        <section class="page-one">
+            <header class="desktop-header">
+                <ul>
+                    <li>Home</li>
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
```

Comment:
- [ ] Kindly make these links, by using the `a` tag where the `href` has the `id` of each section 👍